compiler/grainc/grainc.re

```diff
@@ -57,12 +56,31 @@ let compile_string = name => {
 
   try(
     ignore(
-      Compile.compile_string(~hook=stop_after_typed, ~name, program_str^),
+      {
+        let compileState =
+          Compile.compile_string(~hook=stop_after_typed, ~name, program_str^);
+
+        switch (compileState.cstate_desc) {
```

Comment:
Tiny thing here wrt casing:
```suggestion
        let compile_state =
          Compile.compile_string(~hook=stop_after_typed, ~name, program_str^);

        switch (compile_state.cstate_desc) {
```

Reply:
:-D.  Absolutely.  I laugh because my team can spot my code from my use of snake case where they all use camel, so they've obviously succeeded in their brainwashing! ;)